modules/ocf_ceph/manifests/osd.pp

```diff
@@ -0,0 +1,33 @@
+# Ceph object store daemon configuration
+# Set ocf_ceph::osd::disks to be the disks allocated to ceph
+# If you have issues just dd if=/dev/zero the disk and try again
+
+class ocf_ceph::osd($disks=[]) {
+  include ocf_ceph
+
+  $bootstrap_osd_key = lookup('ceph::bootstrap_osd_key')
+
+  file {
+    default:
+      owner => 'ceph',
+      group => 'ceph';
+    '/var/lib/ceph/bootstrap-osd':
+      ensure => directory;
+    '/var/lib/ceph/bootstrap-osd/ceph.keyring':
```

Comment:
this formatting is a bit weird

Reply:
unsure how it would be improved

Reply:
I think spaces between the different files/directories certainly helps to show they are distinct